python_modules/dagster-graphql/dagster_graphql/test/utils.py

```diff
@@ -170,3 +174,9 @@ def infer_instigation_selector(graphql_context, name):
     selector = infer_repository_selector(graphql_context)
     selector.update({"name": name})
     return selector
+
+
+def infer_resource_selector(graphql_context, name):
+    selector = infer_repository_selector(graphql_context)
+    selector.update({"resourceName": name})
+    return selector
```

Comment:
We shouldn't be in the business of mutating when you could do 

`{**selector, **{"resourceName": name}}`

Immutability eliminates entire classes of programming errors